thrift/Operation.thrift

```diff
@@ -0,0 +1,9 @@
+struct Operation {
+  1: string op,
```

Comment:
does thrift allow enums?

Reply:
Thrift has its own declaration syntax which includes the use of integer ids. The number does not refer to an enum

Reply:
I meant is there any way to turn op into an enum

Reply:
Oh yes, I can change it into an enum with values of GET and PUT. I can look into that

Reply:
Enum for op type added in commit [7a93f91](https://github.com/ySteinhart1/ORTOA/pull/6/commits/7a93f91d7373db753487713a838381c7fad304ec)